cnd/src/http_api/route_factory.rs

```diff
@@ -115,6 +115,20 @@ pub fn create(
         .and(facade.clone())
         .and_then(http_api::routes::index::post_halight_bitcoin_herc20);
 
+    let herc20_hbit = warp::post()
+        .and(warp::path!("swaps" / "herc20" / "hbit"))
```

Comment:
While I see the point of short handing the protocol names in the codebase for readability, I don't agree that we should do it in the api.

To me, it is just makes it harder to use as you would want consistency.
```suggestion
        .and(warp::path!("swaps" / "herc20" / "ethereum" / "erc20" / "han" / "bitcoin" "bitcoin"))
```

Reply:
But, `han` is dead @D4nte :)

`hbit` is not "short" for anything. It is the protocol name and means:

> Locking Bitcoin in an HTLC on the bitcoin blockchain.

We can discuss the names at a later stage, I am not a massive fan but it is good enough for now :)
What is important to me though is that we embrace the concept of "protocol implies asset and ledger". This is gonna make things a lot easier to reason about because of the flat hierachy. Conceptually there is COMIT and beneath are several protocols where each of them is already a concrete concept.

Reply:
> But, han is dead @D4nte :)

Ok.. Where and when was this decision communicated please?

Reply:
We more or less made this decision on the spot, I don't think we ever communicated it explicitly.

Reply:
So, a 2 person spike decision was reverted "on the spot" without further communication to the team?

Reply:
I added you to the thread on bascamp where this was discussed @D4nte :)